pkg/operator/bootstrap.go

```diff
@@ -172,7 +172,5 @@ func (c *Controller) Bootstrap() (*imageregistryapi.ImageRegistry, error) {
 		}
 	}
 
-	addImageRegistryChecksum(cr)
```

Comment:
@legionus this was causing us to consider the initial bootstrap CR to be already processed and ignore it.

Reply:
My PR removes this as well. One more rebase is coming...